docs/book/driver-deployment/installation.md

```diff
@@ -0,0 +1,226 @@
+# vSphere CSI Driver - Installation
+
+The following steps need to be performed on the k8s node where the vSphere CSI drivers controller will be deployed. We recommend installing vSphere CSI driver on your k8s master node.
+
+## Taint Master Node
+
+Make sure that the node is tainted with `node-role.kubernetes.io/master=:NoSchedule`
+
+```cgo
+$ kubectl describe nodes | egrep "Taints:|Name:"
+Name:               <k8s-master-name>
+Taints:             node-role.kubernetes.io/master:NoSchedule
```

Comment:
Can you provide reason why we need to taint master node ?
It is not used in the deployment yaml. instead we use label - https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/manifests/vsphere-7.0/vanilla/deploy/vsphere-csi-controller-deployment.yaml#L25-L26

```
node-role.kubernetes.io/master=
```

Reply:
Yes but we also have these tolerations applied to the driver manifest:

```
      tolerations:
        - operator: "Exists"
          effect: NoSchedule
        - operator: "Exists"
          effect: NoExecute
```

Dropping this comment